### DISCUSSION TITLE
Ordering of lists/where do the bookmarks go?

### DISCUSSION THREAD

**Discussion by Grydpype** [organicmaps/organicmaps#5576] [Q&A] [answered]
Good morning,
possibly a stupid question but I have so far not found an answer:

I have several lists for my bookmarks, so how do I get OM to put the bookmark into the desired list? Or, to put it differently: how does OM decide in which list my bookmark ends up, since I am not asked and cannot choose one?

I am using Version 2023.07.13-10-Google on an Android 10; any suggestions what I am missing here?

Cheers,
Oliver.

**Comment by biodranik** [comment, marked as answer]
Bookmarks are added to the last one used list. If you want to add them to a different list, add a bookmark, move it to another list, then new bookmarks will be added to that list.

**Reply by Grydpype** [reply]
Good morning,
thanks for the quick feedback.

Cheers,
Oliver.
